save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/docker/DockerAgentRunner.kt

```diff
@@ -132,19 +140,27 @@ class DockerAgentRunner(
      */
     @Suppress("UnsafeCallOnNullableType", "TOO_LONG_FUNCTION")
     private fun createContainerFromImage(baseImageId: String,
+                                         pvId: DockerPvId,
                                          workingDir: String,
                                          runCmd: String,
                                          containerName: String,
     ): String {
         val baseImage = dockerClient.findImage(baseImageId, meterRegistry)
             ?: error("Image with requested baseImageId=$baseImageId is not present in the system")
+        val envFileTargetPath = "$SAVE_AGENT_USER_HOME/.env"
         // createContainerCmd accepts image name, not id, so we retrieve it from tags
         val createContainerCmdResponse: CreateContainerResponse = dockerClient.createContainerCmd(baseImage.repoTags.first())
             .withWorkingDir(workingDir)
```

Comment:
Shouldn't it be `envFileTargetPath.substringBeforeLast("/"),` too?

Reply:
No, it's passed from higher levels and its value is `EXECUTION_DIR`